### PR TITLE
ZCS-4796:Create LDAP attribute to turn on or off password reset feature

### PR DESCRIPTION
**Pull request by sneha-patil-synacor**
Added LDAP attributes required for password reset feature

### CODE REVIEW

File: store/conf/attrs/zimbra-attrs.xml [hunk 1]
```diff
@@ -9640,4 +9640,26 @@ TODO: delete them permanently from here
   <globalConfigValue>1518163473</globalConfigValue>
 </attr>
 
+<attr id="3024" name="zimbraFeatureResetPassword" type="boolean" cardinality="single" optionalIn="account,cos,domain" flags="accountCosDomainInherited" since="8.8.9">
```
Comment (rupalid) [inline store/conf/attrs/zimbra-attrs.xml]: Please make it zimbraFeatureResetPasswordEnabled

File: store/conf/attrs/zimbra-attrs.xml [hunk 2]
```diff
@@ -9640,4 +9640,26 @@ TODO: delete them permanently from here
...
+  <defaultCOSValue>FALSE</defaultCOSValue>
+  <desc>whether or not password reset feature is enabled</desc>
+</attr>
+
+<attr id="3025" name="zimbraPrefRecoveryAddress" type="cs_emailp" cardinality="single" optionalIn="account" callback="PrefMailForwardingAddress" since="8.8.9">
```
Comment (rupalid) [inline store/conf/attrs/zimbra-attrs.xml]: Make it zimbraPrefPasswordRecoveryAddress. Also we do not need the PrefMailForwardingAddress attribute

File: store/conf/attrs/zimbra-attrs.xml (hunk 1 above)
Comment (sneha-patil-synacor) [reply store/conf/attrs/zimbra-attrs.xml]: Done

File: store/conf/attrs/zimbra-attrs.xml (hunk 2 above)
Comment (sneha-patil-synacor) [reply store/conf/attrs/zimbra-attrs.xml]: Done

**Pull request closed without merging**